### PR TITLE
feat: update to lnbits 1.0.0

### PR DESCRIPTION
**Pull request by dni**

### CODE REVIEW

File: crud.py [hunk 1]
```diff
@@ -35,109 +33,76 @@ async def delete_lnurlp_settings() -> None:
...
-            data.success_url,
-            data.comment_chars,
-            data.currency,
-            data.fiat_base_multiplier,
-            data.username,
-            data.zaps,
-        ),
+    assert data.wallet, "Wallet is required"
+
+    link = PayLink(
```
Comment (motorina0) [inline crud.py:48]: how about something like:
```py
link = PayLink(**data, **{"id":link_id, "domain": None})
```

**Review by motorina0 — APPROVED** [review]
tested OK
linting requires new lnbits rc

**Comment by bitkarrot** [comment]
whooo hooo. Congrats 1.0.0! LFG!!! :D

**Comment by dni** [comment]
zapping worked :P 
screenshot-1729847435

**Pull request merged**